arches/app/templates/views/components/cards/default-card-report.htm

```diff
@@ -2,8 +2,8 @@
 {% load i18n %}
 
 <div data-bind="css: {'print-hide': !reportExpanded()}">
-    <!-- ko if: showHeaderLine--><hr class="rp-tile-separator"><!-- /ko -->
-    <div class="rp-card-section" data-bind="css: card.model.cssclass, visible: card.fullyProvisional() !== 'fullyprovisional'">
+    <!-- ko if: showHeaderLine && (hideEmptyNodes() === false || ko.unwrap(card.tiles).length > 0)--><hr class="rp-tile-separator"><!-- /ko -->
```

Comment:
`hideEmptyNodes() === false || ko.unwrap(card.tiles).length > 0`

This pattern persists throughout the change. Would it be more straightforward to alter/alter-and-rename the `hideEmptyNodes` val on the card component to use this pattern? That way the logic in the card report stays simple.